e2e_playwright/st_data_editor_config.py

```diff
@@ -79,18 +79,6 @@
     },
 )
 
-
-st.header("Ignore editing-only config options:")
```

Comment:
Hm I see this one deleted. Should this have been deleted and not have been added back?

Reply:
Yep, this was a copy-paste error from before. This isn't relevant for the data editor, only for `st.dataframe`